core/src/main/java/com/klaytn/caver/tx/type/TxTypeCancel.java

```diff
@@ -61,4 +65,39 @@ public static TxTypeCancel createTransaction(
         return Type.CANCEL;
     }
 
+    /**
+     * decode transaction hash from sender to reconstruct transaction with fee payer signature.
+     *
+     * @param rawTransaction RLP-encoded signed transaction from sender
+     * @return TxTypeCancel decoded transaction
+     */
+    public static TxTypeCancel decodeFromRawTransaction(byte[] rawTransaction) {
+        // SenderTxHashRLP = type + encode([nonce, gasPrice, gas, from, txSignatures])
```

Comment:
```suggestion
        // TxHashRLP = type + encode([nonce, gasPrice, gas, from, txSignatures])
```

RLP encoding between TxHashRLP and SenderTxHashRLP is the same, but it would be better to use TxHashRLP.